ui/msg/src/view/scroller.ts

```diff
@@ -16,7 +16,6 @@ class Scroller {
       }),
       { passive: true }
     );
-    window.el = this.element;
```

Comment:
I assume this is just leftover from debugging?

Reply:
oops!